EXILED/Exiled.API/Features/Map.cs

```diff
@@ -84,9 +80,9 @@ DecontaminationController.Singleton.NetworkDecontaminationOverride is Decontamin
         public static ReadOnlyCollection<PocketDimensionTeleport> PocketDimensionTeleports { get; } = TeleportsValue.AsReadOnly();
 
         /// <summary>
-        /// Gets all <see cref="Locker"/> objects.
+        /// Gets all <see cref="MapGeneration.Distributors.Locker"/> objects.
         /// </summary>
-        public static ReadOnlyCollection<Locker> Lockers { get; } = LockersValue.AsReadOnly();
+        public static ReadOnlyCollection<MapGeneration.Distributors.Locker> Lockers { get; } = Features.Lockers.Locker.BaseToExiledLockers.Keys.ToList().AsReadOnly();
```

Comment:
Make this obsolete and reference to lockers' list

Reply:
i am pretty sure the code you made would not work

Reply:
> i am pretty sure the code you made would not work

Yep
![imagen](https://github.com/user-attachments/assets/06e904d5-0bc6-4d33-b29f-a5d07f8347fd)

I need to rename the old property or the new one.